src/bluetooth.py

```diff
@@ -0,0 +1,25 @@
+import pexpect
+import time
+
+import pexpect
+import time
+
+def connect():
```

Comment:
I like the functions but they're still doing the same as the Bash one-liner I mentioned in the other comment.